src/hypertrace/agent/autoinstrumentation/sitecustomize.py

```diff
@@ -0,0 +1,48 @@
+'''Enable instrumentationon all supported modules.''' # pylint: disable=R0401
+import os
+import logging
+from hypertrace.agent import Agent
+
+DEFAULTS = [
+    'flask',
+    'mysql',
+    'postgresql',
+    'grpc:server',
+    'grpc:client',
+    'requests',
+    'aiohttp'
+]
+
+# Initialize logger
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
+
+MODULES = ''
+if 'HT_INSTRUMENTED_MODULES' in os.environ:
+    logger.debug("[env] Loaded HT_INSTRUMENTED_MODULES from env")
+    MODULES = os.environ['HT_INSTRUMENTED_MODULES']
+
+modules_array = MODULES.split(',')
+
+if len(modules_array) == 1 and modules_array[0] == '':
+    modules_array = DEFAULTS
+
+agent = Agent()
+
+for mod in modules_array:
+    if mod is None or len(mod) == 0:
+        continue
+
+    if mod == 'flask':
+        agent.register_flask_app()
+    if mod == 'grpc:server':
```

Comment:
I think this should be a `elif` to avoid evaluating all these conditions all the time.

Reply:
Done.